pallets/creditcoin/src/lib.rs

```diff
@@ -1420,5 +1425,14 @@ pub mod pallet {
 			GATEConract::<T>::put(contract);
 			Ok(())
 		}
+
+		#[transactional]
```

Comment:
```suggestion
```
Transactional is the default since https://github.com/paritytech/substrate/pull/11431 (~May 2022) so it's not needed anymore.

Reply:
#1284